src/Workspaces/SharedUtilitiesAndExtensions/Compiler/VisualBasic/Services/SyntaxFacts/VisualBasicSyntaxFacts.vb

```diff
@@ -218,6 +218,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.LanguageServices
             Return False
         End Function
 
+        Public Function GetStatementOfGlobalStatement(node As SyntaxNode) As SyntaxNode Implements ISyntaxFacts.GetStatementOfGlobalStatement
+            Return Nothing
```

Comment:
```suggestion
            ' GlobalStatement doesn't exist in VB
            Throw New NotImplementedException()
```

Reply:
There are several existing methods I used as an example, e.g.:
https://github.com/dotnet/roslyn/blob/3cca4fdc3b125995bfd32b3a02b5d5c2d2b82504/src/Workspaces/SharedUtilitiesAndExtensions/Compiler/VisualBasic/Services/SyntaxFacts/VisualBasicSyntaxFacts.vb#L522-L525

Reply:
Yes.  I would be ok with another PR followup on this that makes things more consistent.  in this case, as the "Get" methods correspond to cases where the type must be correct (and checked with an IsXXX call before) the right thing for them to do is throw. Thanks :)

Reply:
Ok, now the last question: isn't `InvalidOperationException` better here? The method _is_ implemented, but because the _state_ of the object is VB's Syntax facts, calling this method (_operation_ if you wish :)) is _invalid_.

Reply:
I am fine with either exception type being whatever we standardize on. Thanks.